Dockerfile

```diff
@@ -0,0 +1,57 @@
+# 1: Build the exe
+FROM rust:1.56 as builder
+
+# WORKDIR /usr/src
+
+# RUN USER=root cargo new agora
+
+# COPY Cargo.toml Cargo.lock /usr/src/agora/
+
+WORKDIR /usr/src/agora
+
+# Prepare for static linking
+# RUN apt-get update && \
```

Comment:
For static linking in case we switch to Alpine

Reply:
We can remove this, and worry about static linking when/if we make the switch.